lib/DDG/Spice/FirstRobotics/TeamInfo.pm

```diff
@@ -0,0 +1,46 @@
+package DDG::Spice::FirstRobotics::TeamInfo;
+
+# ABSTRACT: Get info on teams in the FIRST Robotics Competition
+
+use DDG::Spice;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d"; # defaults to this automatically
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'https://www.thebluealliance.com/api/v2/team/frc$1?X-TBA-App-Id=duckduckgo:search-engine:1';
+
+# Obtain the years a team has competed
+spice alt_to => {
+    years_competed => {
+        is_cached => 1,
+        proxy_cache_valid => "200 1d",
+        to => 'https://www.thebluealliance.com/api/v2/team/frc$1/years_participated?X-TBA-App-Id=duckduckgo:search-engine:1'
+    }
+};
+
+triggers any => 'frc', 'frc team', 'first robotics team', 'first robotics competition';
```

Comment:
It may be best to verify that the remainder is just a number, or a number + a keyword like: "location", "motto", "full name" etc

We can do that in a separate PR if you'd like?